### ISSUE TITLE
Remove Site Status link from footers

### ISSUE THREAD

**Issue by diox** [mozilla/addons-server#18883]
Like https://github.com/mozilla/addons-frontend/issues/11308 but in addons-server. Note that there are at least 2 different places with different design that include it in addons-server.

**Comment by Amotul-raheem** [comment]
Hi @diox , I picked up this issue. I started looking into the code base and made the changes required. How do I  go about testing the UI changes? I find it difficult to run locally, Thanks

**Comment by diox** [comment]
Hi,

You'd test that by running it locally :) No need to make an automated test for this change.

**Comment by diox** [comment]
Yes, https://addons-server.readthedocs.io/en/latest/topics/install/docker.html

**Comment by Amotul-raheem** [comment]
thanks, will test locally now.
